### ISSUE TITLE
Create new release with concurrency support

### ISSUE THREAD

**Issue by wassup-** [surpher/PactSwift#120]
Would it be possible to create a new release with concurrency support?

I know I could point to the `main` branch in my SPM dependencies, but this doesn't work in the majority of use-cases where SPM requires stable versions.

**Comment by surpher** [comment]
Yeah, I had that in the back of my mind, but been busy and overseas for a long period of time. 
Will need to fix some tests and clean it up first before it's ready for a decent release. 
It also drops support for Linux in its current state on `main` branch. So that needs a bit of thought and effort put in before a new version is released.

Happy to accept any help cleaning it up.

**Comment by surpher** [comment]
Perhaps some good news @wassup- 

I've taken some time and focused on `PactSwift` and `PactSwiftMockServer`. 
Expect new minor version of `PactSwift - v1.2.0` to hit in early July.

Version `v2.0.0` will definitely be a breaking change as it will change the DSL syntax!

Read more about my plan for `v1.2.0`:
- https://github.com/surpher/PactSwift/discussions/122
- https://github.com/surpher/PactSwift/discussions/109#discussioncomment-9404074

**Comment by surpher** [comment]
@wassup- 
@rastersize

See [v1.2.0](https://github.com/surpher/PactSwift/releases/tag/v1.2.0).

Point your project(s) to `exact: 1.2.0` and see if that works for you. I've run a very shallow sanity check using one of the demo projects and it looks all okay.
